tools/editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -798,6 +798,13 @@ bool TileMapEditor::forward_input_event(const InputEvent& p_event) {
 				canvas_item_editor->update();
 			}
 
+			int tile_under = node->get_cell(over_tile.x, over_tile.y);	
+			String tile_name = "none";
+
+			if (node->get_tileset()->has_tile(tile_under))
+				tile_name = node->get_tileset()->tile_get_name(tile_under);
+			tile_info->set_text("["+String::num(new_over_tile.x)+", "+String::num(new_over_tile.y)+"] Tile: "+tile_name);
```

Comment:
I can't remember why and when, but they held different values. I'll look into this, thanks.